### PR TITLE
Add tests for Cloud Tasks app engine targets

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java11/tasks/pom.xml [hunk 1]
```diff
@@ -45,10 +45,21 @@ Copyright 2019 Google LLC
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-tasks</artifactId>
-      <version>1.6.0</version>
-      <scope>provided</scope>
+      <version>1.3.0</version>
```
Comment (lesv) [inline appengine-java11/tasks/pom.xml]: Why the roll back?

File: appengine-java11/tasks/pom.xml [hunk 2]
```diff
@@ -45,10 +45,21 @@ Copyright 2019 Google LLC
...
     </dependency>
     <!-- [END gae_java11_task_pom] -->
+    <!-- Test dependencies -->
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13-beta-2</version>
```
Comment (lesv) [inline appengine-java11/tasks/pom.xml]: `<scope>test</scope>`

File: appengine-java11/tasks/pom.xml (hunk 1 above)
Comment (averikitsch) [reply appengine-java11/tasks/pom.xml]: The library is broken, hence the added tests to catch this next time.

File: appengine-java11/tasks/pom.xml (hunk 2 above)
Comment (averikitsch) [reply appengine-java11/tasks/pom.xml]: done. Thanks for the review.

**Pull request merged**